lib/api/apiUtils/authorization/prepareRequestContexts.js

```diff
@@ -56,12 +62,18 @@ function prepareRequestContexts(apiMethod, request, sourceBucket,
             objectGetAction, 's3');
         const putRequestContext = generateRequestContext('objectPut');
         requestContexts.push(getRequestContext, putRequestContext);
-        if (apiMethodAfterVersionCheck === 'objectCopy' &&
-        request.headers['x-amz-tagging'] &&
-        request.headers['x-amz-tagging-directive'] === 'REPLACE') {
-            const putTaggingRequestContext =
-                generateRequestContext('objectPutTagging');
-            requestContexts.push(putTaggingRequestContext);
+        if (apiMethodAfterVersionCheck === 'objectCopy') {
```

Comment:
Yes - you can. I added more tests here: https://github.com/scality/Integration/pull/532/files#diff-f0e2239b3e56d3d5e4778cfd2d5d51edR88